src/components/Checkout/Checkout.module.scss

```diff
@@ -77,18 +77,44 @@
   flex-direction: column;
 }
 
+.label {
+  @include small-text;
+  font-size: 12px;
+  color: $c-white;
+  padding-bottom: 4px;
+
+  @include respond-to(tablet) {
+    @include body-text;
+  }
+
+  @include respond-to(desktop) {
```

Comment:
it's not necessary to implement the same media query in desktop, cuz you already have the same on tablet